_api-reference/index-apis/force-merge.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Force merge
+parent: Index APIs
+nav_order: 37
+---
+
+# Force merge
+**Introduced 1.0**
+{: .label .label-purple }
+
+The force merge API operation forces a merge on the shards of one or more indexes. For a data stream, the API forces a merge on the shards of the stream's backing index.
+
+## The merge operation
+
+In OpenSearch, a shard is a Lucene index, which consists of _segments_ (or segment files). Segments store the indexed data. Periodically, smaller segments are merged into larger ones and the larger segments become immutable. Merging reduces the overall number of segments on each shard and frees up disk space. 
+
+## Deleted documents
+
+When a document is deleted from an OpenSearch index, it is not deleted from the Lucene segment but is only marked to be deleted. When the segment files are merged, deleted documents are removed (or _expunged_). Thus, merging also frees up space occupied by documents marked as deleted.
+
+## Force Merge API
+
+In addition to periodic merging, you can force a segment merge using the Force Merge API. 
+
+Use the Force Merge API on an index only after all write requests to the index are completed. Force merge can produce very large segments. If write requests are still sent to the index, then the merge policy does not merge these segments until they primarily consist of deleted documents. This can increase disk space usage and lead to performance degradation.
+{: .warning}
+
+When you call the Force Merge API, the call is blocked until merge completion. If during this time the connection is lost, the force merge operation continues in the background. New force merge requests to the same index will be blocked until the currently running merge operation is complete.
+
+## Force merging multiple indexes
+
+To force merge multiple indexes, you can call the Force Merge API on the following index combinations:
+
+- Multiple indexes
+- One or more data streams containing multiple backing indexes
+- One or more index aliases pointing to multiple indexes
+- All data streams and indexes in a cluster
+
+Force merging multiple indexes executes the merge operation on each shard of a node sequentially. When the force merge operation is in progress, the storage for the shard temporarily increases so all segments can be rewritten into a new segment. When `max_num_segments` is set to `1`, the storage for the shard temporarily doubles.
+
+## Force merging data streams
+
+Force merging data streams is useful for managing a data stream's backing indexes, especially after a rollover operation. Time-based indexes receive indexing requests only during a specified time window. Once that window passes and the index receives no more write requests, you can force merge segments of all index shards into one segment. Searches on single-segment shards are more efficient because they use simpler data structures.
```

Comment:
```suggestion
It can be useful to force merge data streams in order to manage a data stream's backing indexes, especially after a rollover operation. Time-based indexes receive indexing requests only during a specified time period. Once that time period has elapsed and the index receives no more write requests, you can force merge segments of all index shards into one segment. Searches on single-segment shards are more efficient because they use simpler data structures.
```